internal/service/dns_config/model_config_host.go

```diff
@@ -0,0 +1,209 @@
+package dns_config
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
```

Comment:
Was this auto generated? The alias is redundant here

Reply:
removed